### PR TITLE
RDS publishing

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/elifesciences/issues/issues/5434

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Superweird build error on `base` @lsh-0. Know anything about its relationship to Salt and Python versions? First time I see it running with Python 3.

File: jenkinsfiles/Jenkinsfile.rds-publishing [hunk 1]
```diff
@@ -0,0 +1,17 @@
+elifePipeline {
+    stage 'Notify', {
+        elifeSlack "Publishing RDS articles on *prod* (<${env.RUN_DISPLAY_URL}|Build>)", "#deployments"
```
Comment (thewilkybarkid) [inline jenkinsfiles/Jenkinsfile.rds-publishing]: Should be targeting `continuumtest`?

**Comment by lsh-0** [comment]
> Superweird build error on base @lsh-0. Know anything about its relationship to Salt and Python versions? 

Yes indeed. After upgrading the master server yesterday, builder-base formula died immediately upon calling highstate, something it wasn't doing in Vagrant at all. I debugged it on the master server itself and there were Python2-isms lurking in our module code that I [fixed here](https://github.com/elifesciences/builder-base-formula/pull/279). The error in this build is the python2 `base64` module returning it's contents as bytes, rather than a string, that need to be decoded into utf-8.

Before I finished for the day I put the master-server's copy of builder-base on to the fixed branch so highstate on minions wouldn't fail, but it looks like something reverted it. I opened that PR up for your review but will just merge it today so I can get on with things.

I'll kick this build when it's working

**Comment by lsh-0** [comment]
bugs have been fixed but aws credentials appear to be busted, which is similar to the gcs credentials in elife-bot being busted. Will investigate

File: jenkinsfiles/Jenkinsfile.rds-publishing (hunk 1 above)
Comment (giorgiosironi) [reply jenkinsfiles/Jenkinsfile.rds-publishing]: It can be added; `preview` is our main environment so `prod` still need to be included to cater for it.

Comment (giorgiosironi) [reply jenkinsfiles/Jenkinsfile.rds-publishing]: Agreed not to add this, not to slow down the production team using this pipeline.

**Comment by giorgiosironi** [comment]
I see https://github.com/elifesciences/elife-alfred-formula/pull/35 intends to fix the build errors. Going that first.

**Pull request merged**